findoutlie/tests/test_detectors.py

```diff
@@ -14,32 +14,41 @@
 
 MY_DIR = op.dirname(__file__)
 
-# Here you should add the directory containing the findoutlie
-# directory to the Python path.
-# Hint: sys.path
-# Hint: see the solutions if you are stuck.
-# +++your code here+++
-
+sys.path.append("/Users/hugofluhr/code/nipraxis/diagnostics-Lemanics/findoutlie/")
```

Comment:
I should have put relative path here instead of this path that would only work on my machine